fhir-server/src/main/java/com/ibm/fhir/server/util/RestAuditLogger.java

```diff
@@ -368,29 +373,55 @@ public static void logBundle(HttpServletRequest request, Bundle requestBundle, B
                 for (Entry bundleEntry : requestBundle.getEntry()) {
                     if (bundleEntry.getRequest() != null && bundleEntry.getRequest().getMethod() != null) {
                         requestMethod = bundleEntry.getRequest().getMethod();
+                        boolean operation =  bundleEntry.getRequest()
+                                                .getUrl().getValue().contains("$");
```

Comment:
I'm assuming that the URL will always be unencoded here and so its safe to rely on `$` and not `%24`?